debian/ubuntu-advantage-tools.postinst

```diff
@@ -290,10 +290,7 @@ notify_wrong_fips_metapackage_on_cloud() {
 
     fips_metapkg="ubuntu-fips"
 
-    cloud_id=""
-    if command -v "cloud-id" > /dev/null && [ -f /run/cloud-init/instance-data.json ] ; then
-      cloud_id=$(cloud-id)
-    fi
+    cloud_id=$(cloud-id 2>/dev/null || true)
```

Comment:
This byte-by-byte how I thought to implement this, however it still makes an assumption on `cloud-id`: that it won't print anything to stdout on error. If it does, we'll end up setting `cloud_id` to a bad value, like an error message. I'd go for:
```
cloud_id=$(cloud-id 2>/dev/null) || cloud_id=""
```

Reply:
True true, nice catch
I have updated the code